app/src/components/icons.tsx

```diff
@@ -1815,3 +1815,81 @@ export const AllProjectsIcon = () => {
     </svg>
   );
 };
+
+export const AskAiIconOutline = () => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="19"
+      height="23"
+      viewBox="0 0 19 23"
+      fill="none"
+    >
+      <path
+        fillRule="evenodd"
+        clip-rule="evenodd"
```

Comment:
I think this needs to be changed to clipRule to work and not give a client-side error